src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/PropertyPages/DebugPageViewModel.cs

```diff
@@ -880,6 +979,16 @@ private async Task OnLaunchSettingsChanged(ILaunchSettings profiles)
             InitializeDebugTargetsCore(profiles);
         }
 
+        private void InitializeAuthenticationProviders()
+        {
+            _authenticationProviders = new OrderPrecedenceImportCollection<IRemoteAuthenticationProvider>(projectCapabilityCheckProvider: Project);
+            IEnumerable<Lazy<IRemoteAuthenticationProvider, IOrderPrecedenceMetadataView>> providers = Project.Services.ExportProvider.GetExports<IRemoteAuthenticationProvider, IOrderPrecedenceMetadataView>();
```

Comment:
These are already filtered at this point, there's no point in putting these in a OrderPrecedenceImportCollection as GetExports already does this.

Reply:
Thanks, I suspected as much from debugging, but followed the patter in the LaunchType list elsewhere in this class. Will update both.

Reply:
The won't be sorted though, so you will have to do this. I'd consider pulling these objects out into a single component that does all this for you so just pull on one object and that class has all the collections.